.circleci/config.yml

```diff
@@ -34,6 +34,9 @@ jobs:
             pushd samples/js-eventsourced-shopping-cart && npm ci && npm test && popd
             pushd samples/js-valueentity-shopping-cart && npm ci && npm test && popd
 
+            pushd sdk && npm install && sudo npm link && popd
+            pushd samples/valueentity-counter && npm link @lightbend/akkaserverless-javascript-sdk && npm install && npm run build && npm run test && popd
```

Comment:
@andreaTP, any reason for not linking on line 33?

Reply:
no special reasons, I just wanted to add an example without touching the existing structure.